plugins/mcreator-core/blockly/mcreator_blocks.js

```diff
@@ -7,7 +7,7 @@ Blockly.Blocks['event_trigger'] = {
             .appendField(new Blockly.FieldDropdown(
                 jsonToBlocklyDropDownArray(javabridge.getGlobalTriggers())), 'trigger');
         this.setNextStatement(true);
-        this.setStyle('hat_blocks');
+        this.setStyle('hat_blocks');0

```

Comment:
???

Reply:
@KlemenDEV is it a mistake or a Java feature we both don't know?

Reply:
Mistake, but JavaScript probably just "eats" it :D It should be removed